### DISCUSSION TITLE
export an object dictionnary

### DISCUSSION THREAD

**Discussion by Ln92-pixel** [tprouvot/Salesforce-Inspector-reloaded#631] [Q&A] [answered]
Hello,
thank you for this great tool : salesforce inspector reloaded! I 'm looking for a way to export an object with this dictionnary (detail of this fiels with name and type). to clone it. Is it possible to export a data dictionnary for an objet? and if yes, how can i proceed?
thanks in advance for your answer.

**Comment by tprouvot** [comment]
Hi @Ln92-pixel ,
You should be able to export the fields for an sobject by querying the FieldDefinition sobject with the tooling api in data export.
That being said, this is a recurring need and I'm thinking to implement something dedicated to dictionary export

**Reply by Ln92-pixel** [reply]
thank you so much ! I'm also interested by this feature to export easily the object dictionnary

**Reply by tprouvot** [reply, marked as answer]
Here is an example for Account :
`SELECT EntityDefinition.DeveloperName, DeveloperName, DataType, Description FROM FieldDefinition WHERE EntityDefinition.DeveloperName = 'Account'`

With 'Hide Object Columns' enabled and the download button, we are not far from the dictionary export !

<img width="1443" alt="image" src="https://github.com/user-attachments/assets/cc24e5fb-249d-4b5d-9682-acf6689d0131">

**Reply by Ln92-pixel** [reply]
thank you so much Thomas !
